### ISSUE TITLE
Manifest parse errors despite using latest version of Rust

### ISSUE THREAD

**Issue by alexschrod** [rust-secure-code/cargo-geiger#355]
```
❯ cargo build
    Finished dev [unoptimized + debuginfo] target(s) in 0.23s

❯ cargo geiger --version
cargo-geiger 0.11.3

❯ cargo geiger
error: failed to parse manifest at `C:\Path\To\Project\Cargo.toml`

Caused by:
  namespaced features with the `dep:` prefix are only allowed on the nightly channel and requires the `-Z namespaced-features` flag on the command-line
```

Which is odd, because namespaced features has been stable since [1.60](https://blog.rust-lang.org/2022/04/07/Rust-1.60.0.html#new-syntax-for-cargo-features) and none of the other cargo tools I have are choking on my Cargo.toml.

**Comment by pinkforest** [comment]
What does rustup show say and which is used by default ?

**Comment by pinkforest** [comment]
@alexschrod Could you share your Cargo.toml please or the relevant bit re: what it is complaining is fine. Thanks!

**Comment by pinkforest** [comment]
I have a feeling this is some rust target specific thing.. 

I have to investigate but I don't have windows machine so have to figure MS ISO or VM

It works fine for me on both rustup default stable and nightly in Ubuntu and docker

```
$ uname -av
Linux foobar-MS-7C77 5.11.0-46-generic #51~20.04.1-Ubuntu SMP Fri Jan 7 06:51:40 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```
$ rustup show
Default host: x86_64-unknown-linux-gnu
rustup home:  /home/foobar/.rustup

installed toolchains
--------------------

stable-x86_64-unknown-linux-gnu (default)
nightly-2022-01-17-x86_64-unknown-linux-gnu
nightly-x86_64-unknown-linux-gnu
1.46-x86_64-unknown-linux-gnu
1.47-x86_64-unknown-linux-gnu
1.48-x86_64-unknown-linux-gnu
1.49-x86_64-unknown-linux-gnu
1.51-x86_64-unknown-linux-gnu
1.52-x86_64-unknown-linux-gnu
stage1
1.62.1-x86_64-unknown-linux-gnu

installed targets for active toolchain
--------------------------------------

wasm32-unknown-unknown
wasm32-wasi
x86_64-unknown-linux-gnu

active toolchain
----------------

stable-x86_64-unknown-linux-gnu (default)
rustc 1.61.0 (fe5b13d68 2022-05-18)

**Comment by alexschrod** [comment]
Oh, I just noticed something; not sure if it matters, but both on my Windows and my WSL2 Ubuntu installs, I'm getting
```
...
Compiling cargo v0.60.0
...
```
when doing `cargo install cargo-geiger`. Now 1.60 was when the feature was stabilized, so if cargo's version stays in line with that, then it still should've supported it, but it's not 0.63 in any case.

**Comment by pinkforest** [comment]
Oh yeah

**Comment by pinkforest** [comment]
Did you pull the release from crates.io or main from github ? I should really push a release out ! thnaks for reminding :purple_heart:

**Comment by alexschrod** [comment]
I've just done `cargo install cargo-geiger` like the README says. 😅

**Comment by pinkforest** [comment]
yeah it pulls it from the crates.io.. this is my neglect of forgetting to release. I will do one now. Thank you so much :purple_heart:

**Comment by pinkforest** [comment]
I have a feeling we solved #181 as well :partying_face:

**Comment by pinkforest** [comment]
@alexschrod fyi the release is out :partying_face:
